Dockerfile_distroless

```diff
@@ -0,0 +1,19 @@
+FROM  python:3.7-slim AS build
+COPY requirements.txt /tmp/requirements.txt
+RUN set -xe && \
+    echo $(echo BUILD_TIME_ALPINE_VERSION: && /bin/cat /etc/alpine-release) && \
+    pip install -r /tmp/requirements.txt && \
+    mkdir /app 
+COPY ./entrypoint.sh /app/entrypoint.sh
```

Comment:
I suppose entrypoint.sh is not needed anymore. So remove that file and this line please.

Reply:
Done. Removed also other not needed lines